tests/example/index.html

```diff
@@ -20,7 +20,7 @@
     <!-- Cookie Consent plugin -->
     <script src="../../dist/main.js" type="text/javascript"></script>
 
-    <script type="text/plain" data-cookieconsent="statistics">
```

Comment:
This is supposed to test that data-cookieconsent setting works on inline js. If you take the type="text/plain" away then the test will always pass, even if not working.
The test probably needs to be changed so that statistics are accepted as part of the test.